backend/flaskr/utils.py

```diff
@@ -107,18 +110,25 @@ def get_scores_from_database(count=50, countries=[], gamemode="normal"):
     Returns an array of objects representing highscores in the database
     """
     # Construct SQL query
-
```

Comment:
This change would undo the SQL injection fix. Maybe you forgot to sync the most recent changes before making edits?